pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_controller.go

```diff
@@ -143,16 +148,17 @@ func (c *APIReconciler) enqueueAPIResourceSchema(obj interface{}) {
 	for _, obj := range exports {
 		export := obj.(*apisv1alpha1.APIExport)
 		klog.V(2).Infof("Queueing APIExport %s|%s for APIResourceSchema %s", clusterName, export.Name, name)
-		c.enqueueAPIExport(obj)
+		c.enqueueAPIExport(obj, logger, "reason", "APIResourceSchema change", "apiResourceSchema", name)
 	}
 }
 
-func (c *APIReconciler) enqueueAPIExport(obj interface{}) {
+func (c *APIReconciler) enqueueAPIExport(obj interface{}, logger logr.Logger, keysAndValues ...interface{}) {
```

Comment:
add context to the logger before calling this

Reply:
Fix in next commit